tests/AppBundle/Behat/PlatformDemoEnvironmentConstants.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Tests\AppBundle\Behat;
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\InstallType;
+
+class PlatformDemoEnvironmentConstants
```

Comment:
Just an idea: what about inheriting from PlatformEnvConsts, invoking parent constructor and then overriding only the variables that differ? We should do something similar in PageBuilder too.

Advantage: We would avoid defining the same variables (with same values) in many places
Disadvantage: there is a dependency on AdminUI in metarepo - that's ok for me, but needs @damianz5 approval